.github/workflows/test.yml

```
@@ -19,9 +19,9 @@ jobs:
           - 3.8
           - 3.9
         ipfs:
-          - 0.7
           - 0.8
           - 0.9
+          - 0.10
```

YAML parser reads it as a decimal number and removes any trailing zeros:

```suggestion
          - "0.10"
```

---

Good catch, @ibnesayeed! I was just reading about YAML's interpretation of Python 3.10, which would have a similar issue.